Test/Test_counter/CMakeLists.txt

```diff
@@ -0,0 +1,73 @@
+
```

Comment:
Isn't there a way to reduce redundancy of CMaksLists?

Reply:
Non, there is non way.